api/problem/create.py

```
@@ -0,0 +1,40 @@
+# HANDLER -> LOAD
+from website import *
+
+
+#
+#   FUNCTION
+#
+
+# FUNCTION -> DECLARATION
+def output(request: HttpRequest) -> HttpResponse:
+    # DECLARATION -> EXTENSIONS
+    from website.extensions import Response; Response(request)
```

Ruff (0.11.9)

Multiple statements on one line (semicolon)

(E702)